config/routes.rb

```diff
@@ -12,6 +12,7 @@
     namespace :v1, defaults: { format: :json } do
       devise_scope :user do
         resources :topics, only: :index
+        resources :targets, only: %i[create index destroy]
```

Comment:
I added this for error, but I need this for the next pr. If you think that's is an important change, I will fix this.

Reply:
If we want to "simulate" a real life scenario, then these routes should not be here because if this PR gets merged and deploy to production, we are introducing routes that are not working.
Said that, I understand this is a Target project and we don't have a "production" environment or clients that use this app.

Already saw your other PRs so I'm fine with keeping this as it is, just wanted to flag that we should not do this in real projects.

Reply:
I think it'd be better to change this to `resources :targets, only: :create` just like Juli said

Reply:
It fix in last commit